packages/fxa-event-broker/lib/notificationProcessor.ts

```diff
@@ -130,85 +87,93 @@ class ServiceNotificationProcessor {
     this.webhookService.stop();
   }
 
+  private async handleDeleteEvent(message: deleteSchema) {
+    const clientIds = await this.db.fetchClientIds(message.uid);
+    clientIds.forEach(async clientId => {
+      const topicName = this.topicPrefix + clientId;
+      const messageId = await this.pubsub
+        .topic(topicName)
+        .publishJSON({ event: message.event, uid: message.uid });
```

Comment:
What happens if the message isn't sent successfully?

Reply:
Then a processing_error occurs, which logs the error and the message will not be acknowledged to AWS SQS. It will continue retrying.